Makefile.am

```diff
@@ -137,6 +137,11 @@ installconfig: installdir
 		chown $(USER) $(DESTDIR)$(confdir)/sympa.conf; \
 		chgrp $(GROUP) $(DESTDIR)$(confdir)/sympa.conf; \
 	fi
+	@echo "Installing configuration template ..." \
+	echo "installing sympa.conf-dist"; \
+	$(INSTALL) -m 644 -T sympa.conf-dist $(DESTDIR)$(sysconfdir)/sympa.conf-dist; \
+	chown $(USER) $(DESTDIR)$(sysconfdir)/sympa.conf-dist; \
+	chgrp $(GROUP) $(DESTDIR)$(sysconfdir)/sympa.conf-dist;
```

Comment:
sympa.conf will be installed into confdir which may be different from sysconfdir.

Reply:
Yeah, sorry, I didn't remember your [suggestion](https://listes.renater.fr/sympa/arc/sympa-developpers/2019-04/msg00011.html) to use `confdir`.
I changed that for `confdir`, we can make it a controllable option later.

Reply:
I feel good.

Reply:
I created an issue about that: #601